travis/clone_oca_dependencies

```diff
@@ -60,6 +60,11 @@ def git_checkout(deps_checkout_dir, reponame, url, branch):
         command = ['git', 'clone', '-q', url, '-b', branch, checkout_dir]
         _logger.info('Calling %s', ' '.join(command))
         subprocess.check_call(command)
+    else:
+        command = ['git', '--git-dir=' + os.path.join(checkout_dir, '.git'),
```

Comment:
Are you sure about join `.git`? AFAIK it should be just the repo's root directory...